README.md

```diff
@@ -12,7 +18,6 @@ We:
 - **Plan and write blog posts** on or around Hoodie, Hoodie releases, side projects, TGIF, community member profiles, conference and meetup recaps, etc. :memo:
 - **Solicit and curate contributions** to our blog from the larger community. :mag_right:
 - **Plan and manage our Twitter account**, announce new content, help funnel support requests to the engineering team.
-- **Help co-ordinate The Hoodie Letter and Offline First newsletters.**
```

Comment:
I omitted that because I've been curating and coordinating those on my own so they're no longer an Editorial Team responsibility.